header-only/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+cmake_minimum_required(VERSION 3.15...3.24)
+
+project(
+  awkward-headers
+  LANGUAGES CXX
+  VERSION 1.0.0)
```

Comment:
We can version these independently.

Reply:
Independently of `awkward-cpp`, I assume.

Does the version need major, minor, and patch numbers, when the project it's embedded in has much less version-granularity? (Or are the three digits a CMake requirement?)